_pytest/config.py

```diff
@@ -54,7 +54,6 @@ def main(args=None, plugins=None):
             return 4
         else:
             try:
-                config.pluginmanager.check_pending()
```

Comment:
should we remove this check just because we added a new one at a different time in execution?